resources/aws/elb.go

```diff
@@ -66,6 +71,9 @@ func (lb *ELB) CreateOrFail() error {
 	if lb.Client == nil {
 		return microerror.MaskAny(clientNotInitializedError)
 	}
+	if len(lb.PortsToOpen) < 1 {
```

Comment:
I don't know why you like  `< 1` so much, instead of `== 0` :)

But there was already a comment that we should use `== 0` in these cases.